src/app/containers/MediaPage/index.test.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import MediaPage from '.';
+import { shouldShallowMatchSnapshot } from '../../../testHelpers';
```

Comment:
Should we use [Psammead test helpers](https://github.com/bbc/psammead/tree/latest/packages/utilities/psammead-test-helpers)?

If so, then we should avoid shallow snapshots as there was chat around exclusively using [react-testing-library](https://github.com/testing-library/react-testing-library) which doesn't support shallow snapshots for [these reasons](https://kentcdodds.com/blog/why-i-never-use-shallow-rendering).

@pjlee11 what are your thoughts?

Reply:
Switched out the shallow renderer for the other renderer in the same file

Can switch to psammead-helpers if preferred, but that isn't actually being used in simorgh currently

Reply:
I think we ideally don't want to use psammead test helpers in simorgh. I think now this isn't using the shallow render it should be okay